dimscord/objects.nim

```diff
@@ -521,7 +521,7 @@ proc newMessage*(data: JsonNode): Message =
         result.author = newUser(data["author"])
     if "member" in data and data["member"].kind != JNull:
         result.member = some newMember(data["member"])
-    if "referenced_message" in data and data["referenced_message"].kind!=JNull:
+    if "referenced_message" in data and data["referenced_message"].kind != JNull:
```

Comment:
Revert this change, I did `kind!=JNull` on purpose for the sake of not exceeding over 80 characters